k8s/config/prod.sh

```diff
@@ -35,3 +35,4 @@ export GOOGLE_ANALYTICS=UA-49796218-59
 export ACTIVE_ENVIRONMENT=production
 export APP_AUTOMATICALLY_INGEST_CONTENT=True
 export APP_NOTIFY_AFTER_INGESTING_CONTENT=True
+# export APP_TASK_COMPLETION_SURVEY_URL="TO COME"
```

Comment:
Supplying this URL is covered by #1151

Reply:
And this is why this PR can't really go to production yet - there's nothing to link to for now